metadata_mapper/mappers/islandora_mapper.py

```diff
@@ -58,6 +58,7 @@ def to_UCLDC(self):
         # print(f"{self.source_metadata=}")
 
         self.mapped_data = {
+            'calisphere-id': self.legacy_couch_db_id.split('--')[1],
```

Comment:
I'm not exactly sure what the calisphere id should be, but validation requires one. Ideally it should probably match what is currently used for the calisphere.org/item/<item_id> url, but I still don't understand where that comes from when it's not a UUID or an ARK. We should either choose to take this on ourselves as a mapper update we'll have to make at some point across the entire suite of mappers (and remove it from validation requirements), or discuss with CIC.

Reply:
Yeah, we need to do some tracing of code through the old stack to see where this is set. I *think* that if the item doesn't have an ARK, then we use the originating system's id (which in some cases is a UUID).

Reply:
I think this would be a good candidate task for CiC to work on?

Reply:
I've never seen the calisphere/solr id stored in couchdb. You can get the couchdb id from solr, but not the other way around. I think the solr id must currently be created on the sync to solr step (so I don't really think this is within scope for CiC?)